playbooks/README.md

```diff
@@ -79,9 +86,11 @@ $ ansible-playbook -i inventory playbooks/cluster/kubernetes/config.yml
 ### OpenShift cluster
 
 
-Be sure that you have enough space on your machines for docker storage and
-modify [defaults values][docker-storage-setup-defaults] accordingly.
-Follow [docker-storage-setup] documentation for more details.
+Be sure that you have **extra disk** attached on your machines
+for **docker storage** and modify [defaults values][docker-storage-setup-defaults]
+accordingly. In the most of cases you need to set `docker_dev` variable
```

Comment:
No THE required here, simply: In most cases

Reply:
addressed

Reply:
lost in rebase, updated